### ISSUE TITLE
predictor Phase 3: stage ETH/BIWI data, train on real trajectories, and run the frozen comparator

### ISSUE THREAD

**Issue by ll7** [ll7/robot_sf_ll7#5863]
## Current issue-planning review — 2026-09-15

**Audit disposition:** `blocked`. The current audit classifies the issue as blocked; its prerequisite, evidence, or triage gate is incomplete.

**First action:** Check the authorized existing external-data hub before acquiring ETH/BIWI again; verify the exact dataset, license, layout and checksum.

### Ordered implementation / investigation plan

1. Check the authorized existing external-data hub before acquiring ETH/BIWI again; verify the exact dataset, license, layout and checksum.
2. Freeze scene-level splits, bounded epochs, code/lock/thread identity and checkpoint destination under the existing #4013 protocol.
3. Only after staging and current research admission, run the bounded GPU training and register the model with its actual runtime provenance.
4. Run the unchanged learned/CV-MPC/model-free comparator and readiness checks; distinguish real-data empirical output from previous synthetic diagnostic capability.

### Files and owners to inspect

- ROBOT_SF_EXTERNAL_DATA_ROOT registry
- #4013 existing training/checkpoint path
- existing three-planner comparator

### Acceptance and validation

- No training while data or allowed use is unresolved.
- No checkpoint/result claim from synthetic overfit evidence.

### Current dependencies and issue-text corrections

- Dependencies: #4013; #4719; licensed staged ETH/BIWI; approved execution packet
- Replace stale unset-root statements with an actual authorized-host staging check.
- Keep acquisition, training and evaluation receipts separate.
- The supplied draft was checked against the refreshed live issue and complete comment thread; historical rationale is retained below.
- Keep current out-of-scope, ownership, provenance, rights, compute, and claim boundaries intact. Do not close the issue merely because this plan was posted.

### Lifecycle and claim boundary

- Current state labels: `state:blocked-external-input`.
- The issue remains open. No closure, compute submission, release/publication, evidence deletion, or scientific-claim promotion is authorized by this review.
- This review does not request a lifecycle transition; existing assignees, ownership, valid priority labels, and unrelated labels are preserved.
- Proposed non-lifecycle label additions applied with this review: `dependency:has-blockers`.

**Evidence note:** Full live issue body and complete available comment thread were reviewed during the 2026-09-15 refresh. This is planning metadata, not implementation or benchmark evidence; historical measurements remain historical.

Split from #4013 per hotspot triage (2026-07-16): the diagnostic tier there is complete (predictor contract, synthetic training, checkpoint-backed plan(), 3-arm comparator, claim boundary, readiness gate + checksum pinning; acceptance audit #4719 records diagnostic criteria met). This issue carries the REMAINING blocked scope:

1. **Dataset staging (the input)**: real pedestrian trajectories (ETH/BIWI EWAP) staged under ROBOT_SF_EXTERNAL_DATA_ROOT (currently unset; availability: missing). NOTE for the executor: check the private external-data hub FIRST — parts of the ETH data family were acquired and checksum-verified there previously; staging may be a copy+registration, not a download. Licensing: official ETH CVL archive terms; follow the registry skip-if-absent conventions for public CI.
2. **GPU training run**: real-trajectory predictor training (bounded epochs per the #4013 protocol) + checkpoint registration with provenance.
3. **Representative eval**: the pre-built 3-arm comparator (learned vs cv_prediction_mpc vs model-free) over the readiness-gated eval set; report per the existing claim-boundary wording.

Acceptance: dataset registered (checksums), training run provenance-pinned (commit, thread-env, execution context), comparator report artifact, readiness gate green end-to-end. Until (1) is staged this is NOT dispatchable — hence blocked:needs-campaign.

```yaml
schema: goal_autopilot_preparation.v1
repository: ll7/robot_sf_ll7
issue: 5863
source_body_sha256: 12bd250bff3d51a092267de4dd64bd4b2455423f99563b3aa0cb6a1222e2b524
source_comments_sha256: 
audit_schema: open_issue_contract_audit.v1
audit_digest: 1c191411793929640684e0827d763f3a2009a8682ab98dcefa8c2ded3e54f4ea
audit_classification: blocked
next_action: resolve_named_blocker
authority: blocker_owner
execution_mode: blocker
preferred_worker: MaxRunner
expected_pr_runner_label: runner:max
implementation_admitted: False
state_ready_change_proposed: False
mutation_batch: open-issues-20260830-02
```

This packet is preparation evidence only. It never overrides live labels, exact claim state, branch state, typed dependencies, domain gates, compute authority, release authority, or scientific evidence rules.

**Comment by ll7** [comment]
BLOCKED (cheap-lane worker, OpenGateway/auto-smart-routing): issue #5863 is labeled `blocked:needs-campaign` and excluded from cheap-lane dispatch. The required input — staged real pedestrian trajectories (ETH/BIWI EWAP) under ROBOT_SF_EXTERNAL_DATA_ROOT — is absent (env currently unset, availability: missing per the issue body). Remaining scope is GPU training + representative benchmark eval, which the cheap lane forbids (no campaigns, no Slurm, no training runs, no benchmark claims).

Per procedure, no slice is opened. To unblock: stage the ETH/BIWI dataset with checksums (check the private external-data hub for previously acquired/verified parts — may be copy+registration, not download), then re-triage. The diagnostic tier (#4013, audit #4719) is already complete and unaffected.

**Comment by ll7** [comment]
BLOCKED (cheap-lane worker, CommandCode/Tencent-Hy3): re-triage of #5863.

I checked the inputs the issue body asked me to verify before declaring blocked:

- `ROBOT_SF_EXTERNAL_DATA_ROOT` is **unset** in the environment.
- A filesystem scan for any staged ETH/BIWI/UCY `obsmat.txt` or converted tracks under the expected staging dir (`${ROBOT_SF_EXTERNAL_DATA_ROOT}/issue_4013_eth_biwi`) and anywhere under the host found **no dataset bytes** — the private external-data hub copy is not present on this machine.
- The manifest `configs/data/issue_4013_eth_biwi_real_trajectory_manifest.yaml` carries `availability: validated` with a pinned `tree_sha256`, but that is a maintainer-supplied pin for *when the data is staged*; the referenced raw annotations themselves are absent and must be acquired (copy+registration from the hub or a download) and checksum-verified before the contract goes green.

Conclusion: the required input (real pedestrian trajectories staged under `ROBOT_SF_EXTERNAL_DATA_ROOT`) is still **missing**, and the remaining executable scope — GPU real-trajectory training + representative 3-arm comparator eval — is exactly what the cheap lane forbids (no campaigns, no Slurm, no training runs, no benchmark claims). This is the same blocker the issue already carries (`blocked:needs-campaign`) and a prior worker correctly left alone.

Per procedure I open **no PR and no slice**. To unblock: stage the ETH/BIWI EWAP annotations with checksums (check the private external-data hub for the previously acquired/verified copy — likely copy+registration, not a download), then re-triage. The diagnostic tier (#4013, acceptance audit #4719) is complete and unaffected.

**Comment by ll7** [comment]
factory-v2 external blocker: {"blocker": {"category": "external_artifact", "evidence": ["ETH/BIWI EWAP pedestrian-trajectory dataset bytes are not staged anywhere on this host: ROBOT_SF_EXTERNAL_DATA_ROOT is unset, no ewap/biwi/obsmat files found, and standard external-data hub locations are absent. The fail-closed readiness gate (scripts/training/check_issue_4013_real_trajectory_readiness.py --check) reports status='blocked_manifest_contract' with staging_tree.available=false / reason='environment variable unresolved'. This is the first gating artifact: items 2 (GPU training) and 3 (representative eval) cannot even start until it is staged and checksum-validated.", "Even after staging, items 2-3 require a GPU compute campaign and representative benchmark claims, which this lease/cheap lane forbids (authorization compute_submit=false; issue label 'blocked:needs-campaign'). The dataset is the primary machine-checkable blocker; the campaign-lane requirement is a downstream constraint noted here for accuracy."]}, "revival_condition": {"probe": "cd /home/luttkule/git/robot_sf_ll7.worktrees/lease-5863-c150df4060db && bash scripts/dev/run_worktree_shared_venv.sh -- python scripts/training/check_issue_4013_real_trajectory_readiness.py --check", "success_condition": "Report status reaches 'ready_for_real_trajectory_training' (staging_tree.available=true, file_count>0, computed tree_sha256 matches the pinned expected_tree_sha256), i.e. a checksum-validated ETH/BIWI EWAP dataset is staged under a resolved ROBOT_SF_EXTERNAL_DATA_ROOT. After that, route items 2-3 (GPU real-trajectory training + representative 3-arm eval vs cv_prediction_mpc) to a campaign-capable lane rather than the cheap lane."}}

**Comment by ll7** [comment]
### Blocked-queue triage (tracking #7067)



```yaml
blocker_class: external_fact
unblock_condition: >-
  ETH/BIWI EWAP pedestrian-trajectory data is staged under `ROBOT_SF_EXTERNAL_DATA_ROOT` and discoverable to the training path.
watcher: >-
  `test -n "$ROBOT_SF_EXTERNAL_DATA_ROOT"` plus a scan for `obsmat.txt` / converted tracks under that root
next_check_at: >-
  2026-09-01
last_meaningful_progress_at: "2026-07-16"
```

Independently re-verified 2026-07-16: the variable is unset and no EWAP/BIWI/UCY bytes exist on any standard hub location. The fail-closed reason is dataset absence.

Bookkeeping only — no scope, claim, label, or evidence change is made by this comment, and the issue stays open.

**Comment by ll7** [comment]
## Open-issue review — 2026-09-02

**Classification:** external-data- and compute-blocked Phase-3 predictor study.

**Completed background:** #4013 already supplies the predictor contract, synthetic/checkpoint diagnostic, three-arm comparator, checksum/readiness machinery, and conservative claim boundary. This issue owns only the real-trajectory extension.

**Required sequence:** check the private hub first; stage and register the exact ETH/BIWI EWAP bytes with accepted license/provenance and tree checksums; freeze the #4013 training protocol, split, feature schema, epoch/compute cap, and output registry; run GPU training; then evaluate learned versus `cv_prediction_mpc` versus the model-free arm on the existing readiness-gated surface.

**Completion rule:** preserve commit, environment, data/split identities, checkpoint checksum, row accounting, and comparator report. Missing data, schema drift, fallback/degraded rows, or a failed readiness gate stops the study. Real-trajectory training does not by itself establish planner value, real-world prediction validity, or superiority.
